app/models/explainer_item.rb

```diff
@@ -17,7 +17,14 @@ def version_metadata(_changes)
   end
 
   def send_explainers_to_previous_requests(range)
-    TiplineRequest.no_articles_sent(self.project_media_id).where(created_at: Time.now.ago(range.days)..Time.now).find_each do |tipline_request|
+    ids = ProjectMedia.where(id: self.project_media.related_items_ids).map(&:id) # Including child items
```

Comment:
@caiosba, a question, why not use `ProjectMedia.where(id: self.related_items_ids).pluck(:id)` here?

Reply:
@vasconsaurus , yes, we could use `pluck`... it would be faster, actually! I'll make that change.

Reply:
Done (for this line and for a couple of other similar lines). Thanks Manu!